services/static-webserver/client/source/class/osparc/po/Invitations.js

```diff
@@ -101,8 +101,8 @@ qx.Class.define("osparc.po.Invitations", {
 
       const extraCreditsInUsd = new qx.ui.form.Spinner().set({
         minimum: 0,
-        maximum: 1000,
-        value: 0
+        maximum: 199,
+        value: 100
```

Comment:
I would say that these are the credits that one is given when registering.

Q: Who decides this number, the person issuing the invitation? (then this is OK)

Reply:
NK decided. This is used as default.
PO users have access to invitations